module/VuFind/src/VuFind/Controller/BrowseController.php

```diff
@@ -636,11 +636,7 @@ protected function getFacetList(
         $results = $this->getService(\VuFind\Search\Results\PluginManager::class)->get('Solr');
         $params = $results->getParams();
         $params->addFacet($facet);
-        if ($category != null) {
-            $query = $category . ':' . $query;
-        } else {
-            $query = $facet . ':' . $query;
-        }
+        $query = $category != null ? $category . ':' . $query : $facet . ':' . $query;
```

Comment:
I think this can be further simplified to:

```suggestion
        $query = ($category ?? $facet) . ':' . $query;
```